### PR TITLE
Adjust command line parameters

### LINKED ISSUE #61: Add a --help option to wayfire

### PR DESCRIPTION
**Pull request by damianatorrpm**
Adds -v --version.
Adds -h --help.
Adjust --damage-debug /  from -d to -D
Adjust --verbose -v to --debug -d
Also adds a new line for 'unrecognized parameter' / typo

Fixes #61

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -219,31 +237,39 @@ int main(int argc, char *argv[])
...
                 break;
-            case 'v':
+            case 'h':
+                print_help();
+                break;
+            case 'd':
                 log_level = wf::log::LOG_LEVEL_DEBUG;
                 break;
+            case 'v':
+                print_version();
```
Comment (ammen99) [inline src/main.cpp:269]: Would be nice to also print the version in the regular log, line 289.

**Comment by ammen99** [comment]
Just a note, this is a breaking change, `-v` now prints version and `-d` prints debug information.

File: src/main.cpp [hunk 2]
```diff
@@ -260,7 +286,7 @@ int main(int argc, char *argv[])
     signal(SIGABRT, signal_handler);
 #endif
 
-    LOGI("Starting wayfire");
+    LOGI("Starting wayfire", WAYFIRE_VERSION);
```
Comment (soreau) [inline src/main.cpp:289]: You should add a space after 'wayfire', at least, like you did [here](https://github.com/WayfireWM/wayfire/pull/490/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR66). I would suggest something like `LOGI("Starting wayfire version ", WAYFIRE_VERSION);`

**Comment by damianatorrpm** [comment]
I think the test failed due to some unrelated network issue,  do I need to restart it?

**Comment by soreau** [comment]
> I think the test failed due to some unrelated network issue, do I need to restart it?

I wouldn't worry too much about it. Unfortunately, this is common.

**Comment by ammen99** [comment]
I tested this manually, since the builds have gone wrong ... Thanks!

**Pull request merged**